templates/web-application/src/NoMatch.tsx

```diff
@@ -0,0 +1,12 @@
+export interface NoMatchProps {
+    path?: string;
+}
+
+export function NoMatch({ path = "Welcome Home!" }: NoMatchProps) {
```

Comment:
I think this might be a copy paste error. path = "Welcome Home!" doesnt make much sense

Reply:
Oops, totally an artifact following the copy-paste😅